tests/test_docs/helper.py

```diff
@@ -52,3 +52,10 @@ def extract_python_code(file):
         python_str += read_python_file[i]
 
     return python_str
+
+
+def read_md_file(file):
```

Comment:
I would recommend to cal argument `filepath` to be clear that it's not a file instance. Minor comment.